tests/4.5/application_kernels/array_reduce_v1.F90

```diff
@@ -0,0 +1,117 @@
+!===--- array_reduce_v1.F90 ------------------------------------------------===//
+!
+! OpenMP API Version 4.5 Nov 2015
+!
+! This test conducts an array reduction on both the host and device and
+! compares the result. The reduction operator is the default OpenMP add
+! operator over reals. The reduction occurs within a four-level collapsed
+! loop.
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+#define N 768
+
+PROGRAM array_reduce_v1
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_OFFLOADING
+  OMPVV_TEST_SHARED_ENVIRONMENT
+
+  OMPVV_TEST_VERBOSE(test_array_reduce() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+CONTAINS
+  INTEGER FUNCTION test_array_reduce()
+
+    ! xl : mpif90 -O3 -qstrict -qsmp=omp -qoffload array_reduce_v1.F90 -o array_reduce.x
```

Comment:
what is this comment for?

Reply:
I believe this comment was added to record how they compiled the test. I will remove it.